internal/activity.go

```diff
@@ -156,11 +156,13 @@ type (
 	// LocalActivityOptions stores local activity specific parameters that will be stored inside of a context.
 	LocalActivityOptions struct {
 		// ScheduleToCloseTimeout - The end to end timeout for the local activity including retries.
-		// This field is required.
+		// At least on of ScheduleToCloseTimeout or StartToCloseTimeout is required.
+		// defaults to StartToCloseTimeout if not set.
```

Comment:
The default for `ScheduleToCloseTimeout ` for normal activities is infinity. We could be consistent here, but that might be a breaking change.